media-sound/librespot/librespot-0.7.1.ebuild

```diff
@@ -0,0 +1,573 @@
+	tokio-stream-0.1.17
+	tokio-tungstenite-0.27.0
+	tokio-util-0.7.16
+	toml-0.8.23
+	toml_datetime-0.6.11
+	toml_edit-0.22.27
+	tower-0.5.2
+	tower-http-0.6.6
+	tower-layer-0.3.3
+	tower-service-0.3.3
+	tracing-0.1.41
+	tracing-attributes-0.1.30
+	tracing-core-0.1.34
+	try-lock-0.2.5
+	tungstenite-0.27.0
+	typenum-1.18.0
+	uds_windows-1.1.0
+	unicode-ident-1.0.18
+	unicode-width-0.2.1
+	untrusted-0.9.0
+	url-2.5.7
+	utf-8-0.7.6
+	utf8_iter-1.0.4
+	utf8parse-0.2.2
+	uuid-1.18.0
+	vcpkg-0.2.15
+	vergen-9.0.6
+	vergen-gitcl-1.0.8
+	vergen-lib-0.1.6
+	version-compare-0.1.1
+	version-compare-0.2.0
+	version_check-0.9.5
+	walkdir-2.5.0
+	want-0.3.1
+	wasi-0.11.1+wasi-snapshot-preview1
+	wasi-0.14.3+wasi-0.2.4
+	wasm-bindgen-0.2.100
+	wasm-bindgen-backend-0.2.100
+	wasm-bindgen-futures-0.4.50
+	wasm-bindgen-macro-0.2.100
+	wasm-bindgen-macro-support-0.2.100
+	wasm-bindgen-shared-0.2.100
+	web-sys-0.3.77
+	web-time-1.1.0
+	webpki-0.22.4
+	webpki-roots-0.26.11
+	webpki-roots-1.0.2
+	which-4.4.2
+	winapi-0.3.9
+	winapi-i686-pc-windows-gnu-0.4.0
+	winapi-util-0.1.10
+	winapi-x86_64-pc-windows-gnu-0.4.0
+	windows-0.54.0
+	windows-0.61.3
+	windows-collections-0.2.0
+	windows-core-0.54.0
+	windows-core-0.61.2
+	windows-future-0.2.1
+	windows-implement-0.60.0
+	windows-interface-0.59.1
+	windows-link-0.1.3
+	windows-numerics-0.2.0
+	windows-registry-0.5.3
+	windows-result-0.1.2
+	windows-result-0.3.4
+	windows-strings-0.4.2
+	windows-sys-0.45.0
+	windows-sys-0.52.0
+	windows-sys-0.59.0
+	windows-sys-0.60.2
+	windows-targets-0.42.2
+	windows-targets-0.52.6
+	windows-targets-0.53.3
+	windows-threading-0.1.0
+	windows_aarch64_gnullvm-0.42.2
+	windows_aarch64_gnullvm-0.52.6
+	windows_aarch64_gnullvm-0.53.0
+	windows_aarch64_msvc-0.42.2
+	windows_aarch64_msvc-0.52.6
+	windows_aarch64_msvc-0.53.0
+	windows_i686_gnu-0.42.2
+	windows_i686_gnu-0.52.6
+	windows_i686_gnu-0.53.0
+	windows_i686_gnullvm-0.52.6
+	windows_i686_gnullvm-0.53.0
+	windows_i686_msvc-0.42.2
+	windows_i686_msvc-0.52.6
+	windows_i686_msvc-0.53.0
+	windows_x86_64_gnu-0.42.2
+	windows_x86_64_gnu-0.52.6
+	windows_x86_64_gnu-0.53.0
+	windows_x86_64_gnullvm-0.42.2
+	windows_x86_64_gnullvm-0.52.6
+	windows_x86_64_gnullvm-0.53.0
+	windows_x86_64_msvc-0.42.2
+	windows_x86_64_msvc-0.52.6
+	windows_x86_64_msvc-0.53.0
+	winnow-0.7.13
+	wit-bindgen-0.45.0
+	writeable-0.6.1
+	yoke-0.8.0
+	yoke-derive-0.8.0
+	zbus-5.10.0
+	zbus_macros-5.10.0
+	zbus_names-4.2.0
+	zerocopy-0.8.26
+	zerocopy-derive-0.8.26
+	zerofrom-0.1.6
+	zerofrom-derive-0.1.6
+	zeroize-1.8.1
+	zerotrie-0.2.2
+	zerovec-0.11.4
+	zerovec-derive-0.11.1
+	zvariant-5.7.0
+	zvariant_derive-5.7.0
+	zvariant_utils-3.2.1
+"
+
+inherit cargo systemd
+
+DESCRIPTION="An open source client library for Spotify, with support for Spotify Connect"
+# Double check the homepage as the cargo_metadata crate
+# does not provide this value so instead repository is used
+HOMEPAGE="https://github.com/librespot-org/librespot"
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
```

Comment:
The inherit statement was missing in the 0.6.0 ebuild, which broke the live ebuild...